WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -5390,7 +5597,7 @@
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
 				DEFINES_MODULE = YES;
-				ENABLE_BITCODE = YES;
+				ENABLE_BITCODE = NO;
```

Comment:
Yes, it was. The app wasn't compiling, but since the glitch was already fixed in develop, let's revert this. 

cc @astralbodies (thank you both!)